code/modules/clothing/glasses/tajblind.dm

```diff
@@ -0,0 +1,196 @@
+#define MODE_OFF "veiled"
+#define MODE_NATURAL "natural sight"
+#define MODE_CORRECTION "correction"
+
+/obj/item/clothing/glasses/hud/tajblind
+	name = "\improper Tajaran veil"
+	desc = "A sleek, high-tech Tajaran veil, adapted from ancient designs and important to their culture and spirituality.<br>\
+			<span class='info'>Can switch between three modes: Sight-blocking veiled mode, transparent natural sight mode and colorblindness correction mode.</span>"
```

Comment:
Can you move this second line into an `examine()` proc? That's usually how we tend to do extra info bits

Reply:
Look below.
Due to `examine()` already attaching something to every eyewear in the game, using it to attach info here would create user-facing inconsistencies.
If code consistency is a priority, I'll obviously change it.